src/ray/protobuf/common.proto

```diff
@@ -9,6 +9,7 @@ enum Language {
   PYTHON = 0;
   JAVA = 1;
   CPP = 2;
+  NATIVE = 3;
```

Comment:
NATIVE can be ambiguous here, as ray core is implemented in C++ and we probably should treat C++ as native for ray.  How about just `C` to stand for c language?

Reply:
So, as explained in the commit message that introduced this (https://github.com/ray-project/ray/pull/6224/commits/b7eab79609df8142f57285772220575a4b319011), this entry is really just a workaround for #6212. Ultimately it should go away, but in the meantime it is specifically for _any_ compiled binary, regardless of what language it is implemented in. Think of it as "this worker runs native code, so use its executable". The source language for the machine code is not really important.

Reply:
Now, that said, you're right that `NATIVE` could, perhaps, be misconstrued for "Ray native" (i.e., C++). `C` wouldn't really be better sadly, because it, well, doesn't have to be C code. Perhaps `BINARY`?